src/Identity/ApiAuthorization.IdentityServer/src/Configuration/ConfigureSigningCredentials.cs

```diff
@@ -35,13 +35,26 @@ internal class ConfigureSigningCredentials : IConfigureOptions<ApiAuthorizationO
         public void Configure(ApiAuthorizationOptions options)
         {
             var key = LoadKey();
-            options.SigningCredential = key;
+            if (key != null)
+            {
+                options.SigningCredential = key;
+            }
         }
 
         public SigningCredentials LoadKey()
         {
+            // We can't know for sure if there was a configuration section explicitly defined.
+            // Check if the current configuration has any children and avoid failing if that's the case.
+            // This will avoid failing when no configuration has been specified but will still fail if partial data
+            // was defined.
+            if (!_configuration.GetChildren().Any())
```

Comment:
This was a small bug that prevented the key to be just set in code (it would fail if config was not specified). I'll find the related issue and link it to this PR